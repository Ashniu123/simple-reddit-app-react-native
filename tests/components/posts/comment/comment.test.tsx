import * as React from 'react';
import { shallow } from 'enzyme';

import Comment from '../../../../src/components/posts/comment/comment.component';

test('should render Comment component correctly', () => {
	const wrapper = shallow(<Comment />);
	expect(wrapper).toMatchSnapshot();
});