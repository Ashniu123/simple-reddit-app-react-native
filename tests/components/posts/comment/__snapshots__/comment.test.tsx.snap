// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Comment component correctly 1`] = `
<Component
  style={
    Object {
      "borderTopColor": "rgba(0,0,0,0.3)",
      "borderTopWidth": 1,
      "flexDirection": "row",
      "height": 100,
      "marginLeft": 20,
      "padding": 15,
    }
  }
>
  <Component
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      numberOfLines={2}
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
        }
      }
    />
    <Component
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#999",
            "fontSize": 11,
            "fontWeight": "bold",
          }
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#999",
            "fontSize": 11,
            "fontWeight": "bold",
          }
        }
      >
        r/
      </Text>
    </Component>
  </Component>
</Component>
`;
